test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -318,6 +319,8 @@ public static JMXServiceURL getJMXServiceURL() throws MalformedURLException
     @BeforeClass
     public static void setUpClass()
     {
+        ServerTestUtils.daemonInitialization();
```

Comment:
this is to allow `InMemory` class, as I need to setup FileSystem before this function call